pkg/controller/controller.go

```diff
@@ -325,6 +329,31 @@ func getVolumeCapability(
 }
 
 func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.PersistentVolume, error) {
+	if options.PVC.Spec.Selector != nil {
```

Comment:
There is already selector check below.

(this is probably result of rebase)

Reply:
Thanks for catching this. Will remove.